src/components/layout/footer.js

```diff
@@ -117,61 +169,74 @@ const Social = styled.div`
         margin-right: 0.8rem;
     }
     @media ${device.tabletL} {
-        padding: 0;
+        text-align: center;
+        background-color: var(--color-grey-1);
+        display: flex;
+        flex-direction: column;
+        p {
+            padding-top: 3rem;
+            padding: 3rem 0 0.7rem 0;
+        }
+        div {
+            display: flex;
+            justify-content: space-around;
+        }
+    }
+    // **********************************************
```

Comment:
can we remove this too and replace it with a `TODO` comment explaining why it should have display: none

Reply:
Sure